app/views/users/index.html.erb

```diff
@@ -1,29 +1,13 @@
 <h1>Here is the list of Users</h1>
-<article>
-  <h2>User 1</h2>
-  <p>Number of post: 4</p>
-</article>
-<article>
-  <h2>User 2</h2>
-  <p>Number of post: 5</p>
-</article>
-<article>
-  <h2>User 3</h2>
-  <p>Number of post: 3</p>
-</article>
-<article>
-  <h2>User 4</h2>
-  <p>Number of post: 6</p>
-</article>
-<article>
-  <h2>User 5</h2>
-  <p>Number of post: 2</p>
-</article>
-<article>
-  <h2>User 6</h2>
-  <p>Number of post: 1</p>
-</article>
-<article>
-  <h2>User 7</h2>
-  <p>Number of post: 0</p>
-</article>
\ No newline at end of file
+<% @users.each do |user| %>
+<article class='user'>
+  <figure>
+    <img class='user-image' src= '<%= user.photo %>' alt="<%= user.name %>" />
+  </figure>
+  
+  <section class='user-info'> 
+    <h2>User: <%= user.name %></h2>
+    <p>Number of post:<strong> <%= user.post_counter || 0 %> </strong> </p>
+  </section>
```

Comment:
- [ ] Make also sure that when I click on the user it brings me to the user details(users#show page). let's address this also

Reply:
Done